test/fixtures/e2e/amp-form.ssr.html

```diff
@@ -0,0 +1,315 @@
+<!doctype html>
+<html ⚡ allow-viewer-render-template>
+<head>
+    <meta charset="utf-8">
```

Comment:
Can you simplify this document to the minimal markup required to execute the test?

Reply:
sure thing. done.